src/features/courses/course-controller.ts

```diff
@@ -115,7 +116,15 @@ class CourseController {
                 required: false,
                 where: {
                     active: true,
-                }
+                },
+                include: [{
```

Comment:
I'm still not a fan of putting more includes in the default getters we already have

It shouldn't hurt anything but for instance if it isn't an assessment we are doing an extra join we don't need

Reply:
No need to change anything for now though